pkg/kubelet/volume/populator/desired_state_of_world_populator.go

```diff
@@ -102,6 +110,15 @@ func (dswp *desiredStateOfWorldPopulator) populatorLoopFunc() func() {
 	return func() {
 		dswp.findAndAddNewPods()
 
+		if time.Since(dswp.timeOfLastGetPodStatus) < dswp.getPodStatusRetryDuration {
+			glog.Infof( // TODO REDUCE VERBOSITY
```

Comment:
Add a comment to call it out that we do this on purpose to not hitting the runtime frequently.